resources/testdata/glyphs3/WghtVar_NoExport.glyphs

```diff
@@ -401,6 +421,9 @@ key = versionString;
 value = "New Value";
 }
 );
+settings = {
+disablesAutomaticAlignment = 1;
```

Comment:
note that I had to disable automatic component alignment otherwise the "manual-component" composite glyph which uses "hypen" as a component would be automatically modified by Glyphs.app. This feature uses anchors definined in the component (in this case, hyphen) to automatically align the components inside a composite glyph. I didn't want that so I explicitly turned it off, in case you were wondering what this is...

<img width="440" alt="Screenshot 2023-11-09 at 16 02 17" src="https://github.com/googlefonts/fontc/assets/6939968/42eb8653-f005-4405-8f0a-2a147d14ce72">